libafl/src/events/tcp.rs

```diff
@@ -279,6 +279,21 @@ where
                 monitor.display(event.name().to_string(), client_id);
                 Ok(BrokerEventResult::Handled)
             }
+            Event::UpdateItemGeometry {
```

Comment:
@andreafioraldi is this fine or should it just be `UpdateUserStats`?

Reply:
yeah I think UpdateUserStats is better

Reply:
<img width="1149" alt="image" src="https://github.com/AFLplusplus/LibAFL/assets/14891443/31bd11bc-e285-4800-b578-f75b565b0cf3">

Is Here `stability` the same metric as AFL‘s? 
If yes, UpdateUserStats is better.

Reply:
yes